assets/src/js/helper.js

```diff
@@ -40,3 +40,36 @@ function dokan_get_i18n_date_format( format = true ) {
 
   return datepickerFormat;
 }
+
+/**
+ * Dokan Sweet Alert
+ *
+ * @since DOKAN_LITE
+ *
+ * @param {string} message The event message for notification
+ * @param {object} [options] Configuration of sweet alert.
```

Comment:
Kindly write {object} [options] to elaborate which parameters are expected in the object.